sdk/tests/graph/test_ecs.py

```diff
@@ -0,0 +1,83 @@
+#
+# Bentobox
+# SDK - Graph
+# Graph ECS
+#
+
+from bento.graph.ecs import *
+
+
+def test_graph_ecs_entity():
+    components = ["position"]
+    entity = GraphEntity(components=components)
+    # check Entity's components accessible via `.components`
+    assert len(set([c._name for c in entity.components]) - set(components)) == 0
```

Comment:
That would not work. Consider:
- `[c._name for c in entity.components]` evaluates to `[a,b,c]`
- `components` evaluates to `[c,a,b]`

the assertion would fail.

Reply:
Yeah you're right. +1 for removing the square brackets too